test/api/controllers/collectionController.test.js

```diff
@@ -295,6 +371,78 @@ describe('Test: collection controller', () => {
 
   describe('#validateSpecifications', () => {
     it('should call the right functions and respond with the right response', () => {
+      request.input.resource.index = 'myindex';
+      request.input.resource.collection = 'mycollection';
+      request.input.body = {
+        strict: true,
+        fields: {
+          myField: {
+            mandatory: true,
+            type: 'integer',
+            defaultValue: 42
+          }
+        }
+      };
+
+      CollectionController.__set__({
+        createSpecificationList: sinon.stub().resolves({
+          _id: 'indexcollection',
+          _source: {
+            validation: 'validation',
+            index: 'index',
+            collection: 'collection'
+          }
+        }),
+        validateSpecificationList: sinon.stub().resolves({valid: true})
```

Comment:
You should test that this stub is called with the good arguments. (Which is not the case here)